src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -601,4 +601,49 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
 
     def encodeJson(t: GetStorageRootResponse): JValue = encodeAsHex(t.storageRoot)
   }
+
+  def extractStorageKeys(input: JValue): Either[JsonRpcError, Seq[StorageProofKey]] = {
+    import cats.syntax.traverse._
+    import cats.syntax.either._
+    input match {
+      case JArray(elems) =>
+        elems.traverse { x =>
+          extractQuantity(x)
+            .map(StorageProofKey.apply)
+            .leftMap(_ => InvalidParams(s"Invalid param storage proof key: $x"))
+        }
+      case _ => Left(InvalidParams())
+    }
+  }
+
+  implicit val eth_getProof =
```

Comment:
nitpick: do you mind adding an explicit type here? implicits behave a bit less frantic when those are used.